win32ss/gdi/ntgdi/font.h

```diff
@@ -40,13 +40,8 @@ typedef struct _FONT_ASPECT
     WORD RenderMode;
```

Comment:
The EMULATION_BOLD_ITALIC + that anonymous union above could be at least simplified as (keeping definition within FONT_ASPECT):
```c
    _ANONYMOUS_UNION union {
        struct {
            BYTE Bold;
            BYTE Italic;
        } Emu;
        WORD EmuBoldItalic;
    } DUMMYUNIONNAME;
```
as you don't seem to use the "EMULATION_BOLD_ITALIC" type alone anywhere else.

Reply:
OK.